### ISSUE TITLE
Error creating .hyper.css

### ISSUE THREAD

**Issue by JakeHP** [chrisdothtml/hyper-stylesheet#1]
Hyper 1.0.0
No other packages installed
Ubuntu 16.04 64 bit

```hpm install hyper-stylesheet```

```
remote.js:217 Uncaught Error: Could not call remote function ''. Check that the function signature is correct. Underlying error: ENOENT: no such file or directory, open '/home/jake/.hyper.css'
Error: Could not call remote function ''. Check that the function signature is correct. Underlying error: ENOENT: no such file or directory, open '/home/jake/.hyper.css'
    at callFunction (/opt/Hyper/resources/electron.asar/browser/rpc-server.js:228:11)
    at EventEmitter.<anonymous> (/opt/Hyper/resources/electron.asar/browser/rpc-server.js:316:5)
    at emitMany (events.js:127:13)
    at EventEmitter.emit (events.js:201:7)
    at WebContents.<anonymous> (/opt/Hyper/resources/electron.asar/browser/api/web-contents.js:228:13)
    at emitTwo (events.js:106:13)
    at WebContents.emit (events.js:191:7)metaToValue @ remote.js:217__WEBPACK_IMPORTED_MODULE_7__utils_config__.subscribe @ bundle.js:51915emitOne @ events.js:96emit @ events.js:188
```

After I manually created the file myself then Hyper atleast loads, but no `Stylesheet` dropdown appears (and all styles are lost for hyper completely, as expected since stylesheet file is empty). Lastly, just an informational bit, I've installed many other packages and everything works as expected. Let me know if there's any other info I can provide to help. :+1:

**Comment by chrisdothtml** [comment]
@Jakehp Just to make sure, if you remove hyper-stylesheet from your plugins, everything works normally, right?

**Comment by JakeHP** [comment]
Yep, no issues once removed.

**Comment by chrisdothtml** [comment]
It's a bit difficult to debug custom builds of hyper plugins since [it doesn't support npm link'd plugin modules](https://github.com/zeit/hyper/issues/498). However, I'm thinking that it might for some reason be the file watcher that's causing issues.

Try modifying `~/.hyper-plugins/node_modules/hyper-stylesheet/lib/init.js`, and comment out [line #36](https://github.com/chrisdothtml/hyper-stylesheet/blob/d7b5705ed66edd0ce5e6a1bdd72026823858d82a/lib/init.js#L36). Once you've done that, hard refresh or just restart hyper, and see if at least the Stylesheet nav button appears.

**Comment by Donaldini** [comment]
I'm getting the same results.
Removing the plugin from _.hyper.js_ solves the issue and starts Hyper without issues.
Else I cannot even start Hyper again.

But if I install it, see the error, change the _ini.js_ file and hard reload I get a blackened out Hyper as seen here:

![2018-02-01 14_14_30-](https://user-images.githubusercontent.com/2185680/35680308-405ee350-075a-11e8-980e-bb5b0084f58d.png)

**Comment by chrisdothtml** [comment]
@Donaldini this plugin is being buggy for me now even on Mac. I'll have to toy around with it and see where the issue is happening (I just discovered there's [an actual way to debug plugins](https://github.com/zeit/hyper/issues/2568#issuecomment-355227246), so that's nice)

**Comment by chrisdothtml** [comment]
@Jakehp @Donaldini this should now be fixed in `v0.1.1`

**Comment by Donaldini** [comment]
Can confirm that it is fixed, thanks!
